backend/config/corpora-api.yml

```diff
@@ -412,6 +412,66 @@ paths:
                     type: integer
         "400":
           $ref: "#/components/responses/400"
+    post:
+      tags:
+        - collections
+      summary: Create a collection
+      security:
+        - cxguserCookie: []
+      description: >-
+        Creates a new collection in corpora. Once a collection is created it can be linked to datasets.
+      operationId: corpora.lambdas.api.v1.collection.create_collection
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - name
+                - description
+                - contact_name
+                - contact_email
+              properties:
+                name:
+                  type: string
+                  description: name of the collecton
+                description:
+                  type: string
+                  description: description of the collection
+                contact_name:
+                  type: string
+                  description: name of the primary person of contact for the collection
+                contact_email:
+                  type: string
+                  description: email of contact person for the collection
+                data_submission_policy_version:
+                  type: string
+                  description: Which data submission policy was the collection submitted under
+                links:
```

Comment:
I think there is link datatype you can reference, rather than duplicating it here.